packages/flutter/test/widgets/app_overrides_test.dart

```diff
@@ -26,12 +26,17 @@ class TestRoute<T> extends PageRoute<T> {
 }
 
 Future<Null> pumpApp(WidgetTester tester) async {
-  await tester.pumpWidget(new WidgetsApp(
-    color: const Color(0xFF333333),
-    onGenerateRoute: (RouteSettings settings) {
-      return new TestRoute<Null>(settings: settings, child: new Container());
-    },
-  ));
+  await tester.pumpWidget(
+    new Directionality(
```

Comment:
It turns out that this hid some real bugs. WidgetsApp is the widget that's supposed to introduce the Directionality. :-)

I'll post a fix.

Reply:
Thanks :)